napari/layers/base/base.py

```diff
@@ -615,61 +619,23 @@ def _is_moving(self, value):
             assert self._moving_coordinates is not None
         self._private_is_moving = value
 
-    @property
-    def _dims_displayed(self):
```

Comment:
Removing `_ndisplay`, `_dims_displayed`, and `_dims_not_displayed` causes a lot of the deltas in this PR. I'd be willing to bring these back to reduce the size of this PR and make references to these things slightly more concise.

Reply:
I don't think this is necessary, this will be a nice clean break

Reply:
I agree with @alisterburt, keeping them around is more confusing than not.